beaker/client.py

```diff
@@ -200,7 +201,7 @@ def _make_session(self) -> requests.Session:
             backoff_factor=1,
             status_forcelist=self.RECOVERABLE_SERVER_ERROR_CODES,
         )
-        session.mount("https://", HTTPAdapter(max_retries=retries))
+        session.mount("https://", HTTPAdapter(max_retries=retries, pool_maxsize=(os.cpu_count() or 16) * 2))
```

Comment:
Might be good to cap this at a reasonable number:
```suggestion
        session.mount("https://", HTTPAdapter(max_retries=retries, pool_maxsize=max((os.cpu_count() or 16) * 2), 32))
```

Reply:
I don't know. If you have 64 cores, you'll get the warning again if we do that. I don't think it's such a big problem to have many connections hanging open? The server can close them too, right?

Reply:
We'll give it a try